utilities/cli/version_check.py

```
@@ -0,0 +1,71 @@
+    print(f"Current:  {local_hash[:8] if local_hash else '<unknown>'} ({commit_file})")
+    print(f"Expected: {cli_pinned_commit[:8]}")
+    print("To reinstall pinned CLI, run:")
+    print(f"  CLI_FORCE_UPDATE=1 {cmd_name} --help")
+    print("════════════════════════════════════════════════════")
+
+
+if __name__ == "__main__":
+    try:
+        check_for_cli_updates()
+    except Exception:
+        pass  # Suppress all exceptions to ensure CLI always runs, even if version check fails.
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)